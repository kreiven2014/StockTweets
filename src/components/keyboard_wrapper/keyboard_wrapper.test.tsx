/* REACT */
import React from "react";

/* MODULES */
import renderer from "react-test-renderer";

/* CUSTOM MODULES */
import KeyboardWrapper from "./keyboard_wrapper";
import TextInput from "src/components/base_input/base_input";

describe("KeyboardWrapper tests", () => {
  test("renders correctly", (): void => {
    const tree = renderer
      .create(
        <KeyboardWrapper>
          <TextInput
            placeholder="placeholder"
            value="value"
            // FUNCTIONS
            onChangeText={(value: string) => { }} // eslint-disable-line @typescript-eslint/no-unused-vars
          />
        </KeyboardWrapper>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
