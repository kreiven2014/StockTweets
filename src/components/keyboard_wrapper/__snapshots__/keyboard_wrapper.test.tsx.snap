// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KeyboardWrapper tests renders correctly 1`] = `
<View
  onLayout={[Function]}
  style={
    Array [
      Array [
        Object {
          "flex": 1,
        },
        undefined,
      ],
      Object {
        "paddingBottom": 0,
      },
    ]
  }
>
  <View
    style={
      Object {
        "width": "100%",
      }
    }
  >
    <TextInput
      allowFontScaling={false}
      onChangeText={[Function]}
      placeholder="placeholder"
      rejectResponderTermination={true}
      style={
        Array [
          Object {
            "borderColor": "#D7D9E0",
            "borderRadius": 4,
            "borderWidth": 1,
            "flex": 1,
            "fontSize": 24,
            "height": 68.5,
            "paddingHorizontal": 24,
          },
          undefined,
          undefined,
          Object {
            "borderColor": "#D7D9E0",
          },
        ]
      }
      underlineColorAndroid="transparent"
      value="value"
    />
  </View>
</View>
`;
